scripts/lib/build-utils.mjs

```diff
@@ -301,6 +318,8 @@ export default {
       tasks.push(() => this.buildNerdctlStub('linux'));
     }
     tasks.push(() => this.buildRdCtl(os.platform()));
+    tasks.push(() => this.buildVtunnel('windows'));
```

Comment:
The function checks for `win32`, so it should be passing that in instead of `windows`. (Also, our resource directory is named `win32`.)

This also means there's an opportunity to clean up the nerdctl build method.